spec/lib/farscape/cache_spec.rb

```diff
@@ -11,10 +11,6 @@
       expect(Farscape.cache).to be_a(ActiveSupport::Cache::MemoryStore)
     end
 
-    it 'does not require the full active_support suite' do
-      expect(Hash.new).not_to respond_to(:slice)
```

Comment:
Can replace this assertion with `expect($".grep /active_support.rb/).not_to be_any`, which will only fail if we require the whole thing.